viper/modules/pehash/pehasher.py

```diff
@@ -6,30 +6,11 @@
 
 import bz2
 import hashlib
-
-try:
```

Comment:
@Rafiot I recall now why I changed this: 
 * `calculate_pehash` is only ever called from pe.py - which itself already checks whether `pehash` [is availble](https://github.com/Rafiot/viper/blob/production/viper/modules/pe.py#L922) and prints a dependency message for the user if not.
* pehasher.py uses `self.log` without being a Class or having `self.log` - thus it would fail/crash if this code would be reachable in any way.